SWIG/swap.i

```diff
@@ -439,11 +439,26 @@ class MakeOIS {
         MakeOIS& withEffectiveDate(const Date&);
         MakeOIS& withTerminationDate(const Date&);
         MakeOIS& withRule(DateGeneration::Rule r);
+        MakeOIS& withFixedLegRule(DateGeneration::Rule r);
+        MakeOIS& withOvernightLegRule(DateGeneration::Rule r);
```

Comment:
These should also be added to the Python `MakeOIS` function below

Reply:
Works for me.  Thanks!

Reply:
Done.